File: domain/src/main/java/io/prizy/domain/contest/model/Merchant.java
package io.prizy.domain.contest.model;

import lombok.Builder;

/**
 * @author Nidhal Dogga
 * @created 01/03/2021 21:18
 */

@Builder
public record Merchant(
  String name,
  String siren,
  Float capital,
  String address,
  String logoAssetId
) {
}

File: domain/src/main/kotlin/io/prizy/domain/contest/model/Merchant.kt
package io.prizy.domain.contest.model

/**
 *  @author Nidhal Dogga
 *  @created 01/03/2021 21:18
 *  @credits prizy.io
 *  All rights reserved
 */

data class Merchant(
  val name: String,
  val siren: String,
  val capital: Float,
  val address: String,
  val logoMediaId: String
)
